session/session.go

```diff
@@ -520,17 +522,22 @@ func chProposalNotif(parts []string, curr string, chProposal *pclient.BaseChanne
 }
 
 // SubChProposals implements sessionAPI.SubChProposals.
-func (s *Session) SubChProposals(notifier perun.ChProposalNotifier) error {
-	s.Debug("Received request: session.SubChProposals")
+func (s *Session) SubChProposals(notifier perun.ChProposalNotifier) perun.APIErrorV2 {
+	s.WithField("method", "SubChProposals").Info("Received request: session.SubChProposals")
```

Comment:
Since the method names is logged as a field, you could ignore it in the log message. Instead you could include the parameters. See line line 147 (AddPeerID) for reference.

Reply:
I have added a suggestion to this line, so that we use uniform error/info messages across all the APIs.